### PR TITLE
MainPage

### PR DESCRIPTION
**Pull request by Aliaksandra-Leanovich**

### CODE REVIEW

File: src/components/AdvantagesSection/AdvantagesSection.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          </DecriptionSC>
+        </InformationSC>
+        <AdvantagesSC>
+          {(
+            t("advantages", {
+              returnObjects: true,
+              defaultValue: [],
+            }) as []
+          ).map((item: itemProps, index) => (
+            <AdvantageSC key={index}>
```
Comment (shumak-yaroslav) [inline src/components/AdvantagesSection/AdvantagesSection.tsx]: don't use index for key

Снимок экрана 2023-02-14 в 2 34 42 PM

File: src/components/AdvantagesSection/AdvantagesSection.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          <DecriptionSC>
+            <TitleSC>{t("advantages.title")}</TitleSC>
+            <TextSC>{t("advantages.decription")}</TextSC>
+            <LinkSC>
+              <LinkTextSC>{t("advantages.link")}</LinkTextSC>
+              <ArrowImage />
+            </LinkSC>
+          </DecriptionSC>
+        </InformationSC>
+        <AdvantagesSC>
```
Comment (shumak-yaroslav) [inline src/components/AdvantagesSection/AdvantagesSection.tsx]: separate component

File: src/components/AdvantagesSection/style.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  row-gap: 16px;
+
+  width: 100%;
+  max-width: 770px;
+`;
+
+export const ImageSC = styled.div`
+  width: 100%;
+  max-width: 784px;
+`;
```
Comment (shumak-yaroslav) [inline src/components/AdvantagesSection/style.ts:66]: separator

File: src/components/AdvantagesSection/style.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+export const TitleSC = styled.p`
+  font-size: 32px;
+  line-height: 34px;
+`;
+export const TextSC = styled.p`
+  font-size: 16px;
+  line-height: 18px;
+`;
+export const LinkSC = styled.div`
```
Comment (shumak-yaroslav) [inline src/components/AdvantagesSection/style.ts:88]: same

File: src/components/Footer/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  SocialsSC,
+  TextSC,
+  TopSC,
+  WrapperSC,
+} from "./style";
+import { IItem } from "./types";
+import Logo from "/public/icons/logo.svg";
+import Payments from "/public/icons/payments.png";
+
+export const Footer = () => {
```
Comment (shumak-yaroslav) [inline src/components/Footer/Footer.tsx:6]: break down into separate components

File: src/components/Footer/style.ts [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+  display: flex;
+  column-gap: 200px;
+`;
+
+export const BlockFirstSC = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+export const BlockWithLogoSC = styled.div`
```
Comment (shumak-yaroslav) [inline src/components/Footer/style.ts]: separator

File: src/components/Footer/style.ts [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+  display: flex;
+  flex-direction: column;
+  row-gap: 16px;
+`;
+export const LogoSC = styled.img`
+  width: 100%;
+  max-width: 194px;
+`;
+export const PhoneSC = styled.p`
```
Comment (shumak-yaroslav) [inline src/components/Footer/style.ts]: same

File: src/components/Footer/style.ts [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+`;
+
+export const BlockFirstFooterSC = styled.div`
+  font-family: "Roboto";
+
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+
+  color: #b7b7b7;
```
Comment (shumak-yaroslav) [inline src/components/Footer/style.ts]: create separate file with colors constants

File: src/components/Footer/style.ts [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+  display: flex;
+  flex-direction: column;
+  row-gap: 16px;
+`;
+
+export const PaymentSC = styled.img`
+  width: 100%;
+  max-width: 604px;
+`;
+export const TextSC = styled.p`
```
Comment (shumak-yaroslav) [inline src/components/Footer/style.ts]: separator

File: src/components/Footer/style.ts [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+  font-family: "Roboto";
+
```
Comment (shumak-yaroslav) [inline src/components/Footer/style.ts]: why do we need this empty string?

File: src/components/FormSection/FormSection.tsx [hunk 11]
```diff
@@ -0,0 +1,44 @@
+import { t } from "i18next";
+
+import React from "react";
+import { useTranslation } from "react-i18next";
+import Image from "../../../public/icons/formImage.svg";
```
Comment (shumak-yaroslav) [inline src/components/FormSection/FormSection.tsx]: absolute path

File: src/components/FormSection/FormSection.tsx [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+export const FormSection = () => {
+  const { t } = useTranslation();
+  const placeholder = t("form.input");
+  return (
+    <WrapperSC>
+      <ContainerSC>
+        <ImageSC>
+          <Image />
+        </ImageSC>
+
```
Comment (shumak-yaroslav) [inline src/components/FormSection/FormSection.tsx]: why do we need this empty string?

File: src/components/FormSection/FormSection.tsx [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+        <BlockSC>
+          <InformationSC>
+            <TitleSC>{t("form.title")}</TitleSC>
+            <TextSC>{t("form.text")}</TextSC>
+          </InformationSC>
+
```
Comment (shumak-yaroslav) [inline src/components/FormSection/FormSection.tsx]: same

File: src/components/FormSection/style.ts [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+`;
+
+export const ContainerSC = styled.div`
+  padding: 76px 120px 76px 430px;
+  position: relative;
+`;
+export const BlockSC = styled.div`
```
Comment (shumak-yaroslav) [inline src/components/FormSection/style.ts:34]: separator

File: src/components/FormSection/style.ts [hunk 15]
```diff
@@ -0,0 +1,88 @@
...
+export const TitleSC = styled.p`
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 53px;
+`;
+export const TextSC = styled.p`
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 29px;
+
```
Comment (shumak-yaroslav) [inline src/components/FormSection/style.ts:66]: delete this

File: src/components/Header/Header.tsx [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+} from "./style";
+import Logo from "/public/icons/logo.svg";
+
+export const Header = () => {
+  return (
+    <StyledHeaderSC>
+      <WrapperSC>
+        <ContainerTopSC>
+          <InformationSC>
+            <PhoneNumberSC>8 495 259 25 00</PhoneNumberSC>
```
Comment (shumak-yaroslav) [inline src/components/Header/Header.tsx]: make const `defaultPhoneNumber`

File: src/components/MainSection/MainSection.tsx [hunk 17]
```diff
@@ -0,0 +1,30 @@
...
+  DescroptionSC,
+  TitleSC,
+  WrapperSC,
+} from "./style";
+import BackgroundImage from "/public/icons/mainBackground.svg";
+import { useTranslation } from "react-i18next";
+
+export const MainSection = () => {
+  const { t } = useTranslation();
+  return (
```
Comment (shumak-yaroslav) [inline src/components/MainSection/MainSection.tsx:17]: separator

File: src/components/Navigation/Navigation.tsx [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+import CartIcon from "/public/icons/cart.png";
+import ProfileIcon from "/public/icons/profile.png";
+import { ILink } from "./types";
+
+export const Navigation = () => {
+  const { t } = useTranslation();
+
+  return (
+    <StyledNavigationSC>
+      <LinksContainerSC>
```
Comment (shumak-yaroslav) [inline src/components/Navigation/Navigation.tsx]: separate component

File: src/components/NewProductsSection/NewProductsSection.tsx [hunk 19]
```diff
@@ -0,0 +1,91 @@
...
+import { ContainerSC, TitleSC, WrapperSC, SliderSC } from "./style";
+import { useTranslation } from "react-i18next";
+import Product1Image from "/public/icons/product1.svg";
+import Product2Image from "/public/icons/product2.svg";
+import Product3Image from "/public/icons/product3.svg";
+import Product4Image from "/public/icons/product4.svg";
+import Product5Image from "/public/icons/product5.svg";
+import Product6Image from "/public/icons/product6.svg";
+
+const data = [
```
Comment (shumak-yaroslav) [inline src/components/NewProductsSection/NewProductsSection.tsx:12]: separate file with constants

Comment (Aliaksandra-Leanovich) [reply src/components/NewProductsSection/NewProductsSection.tsx:12]: we cannot move it cuz there are tsx component

File: src/components/SaleSection/SaleSection.tsx [hunk 20]
```diff
@@ -0,0 +1,117 @@
...
+  },
+  {
+    name: "R&B Black Food 3.7",
+    type: "Бальзам-ополаскиватель для волос с экстрактами 3 черных семян и 7 черных плодов",
+
+    image: <ProductImage />,
+  },
+];
+
+export const SaleSection = () => {
```
Comment (shumak-yaroslav) [inline src/components/SaleSection/SaleSection.tsx:60]: break down into separate components

File: src/components/SaleSection/style.ts [hunk 21]
```diff
@@ -0,0 +1,149 @@
...
+  position: relative;
+`;
+
+export const CustomSwiperSC = styled(Swiper)`
+  .swiper {
+    width: 100%;
+    display: flex;
+    align-self: right;
+  }
+  .fullscreen .swiper-slide .backdrop {
```
Comment (shumak-yaroslav) [inline src/components/SaleSection/style.ts:47]: separator between selectors

File: src/components/SliderMeduim/SliderMedium.tsx [hunk 22]
```diff
@@ -0,0 +1,82 @@
...
+  NameSC,
+  NewPriceSC,
+  OldPriceSC,
+  PriceSC,
+  TypeSC,
+} from "./style";
+import { IData } from "./types";
+import ArrowImage from "/public/icons/arrow.svg";
+
+export const SliderMedium = ({ data }: IData) => {
```
Comment (shumak-yaroslav) [inline src/components/SliderMeduim/SliderMedium.tsx:23]: break down into separate components

File: src/i18n/i18n.js [hunk 23]
```diff
@@ -0,0 +1,31 @@
...
+  ru: {
+    translation: translationsRU,
+  },
+};
+const DETECTION_OPTIONS = {
+  order: ["navigator"],
+};
+
+i18n
+  .use(initReactI18next) // passes i18n down to react-i18next
```
Comment (shumak-yaroslav) [inline src/i18n/i18n.js]: delete comments

File: src/i18n/i18n.js [hunk 24]
```diff
@@ -0,0 +1,31 @@
...
+  .use(LanguageDetector)
+  .init({
+    resources,
+    fallbackLng: "ru", // default language
+    keySeparator: false, // we do not use keys in form messages.welcome
```
Comment (shumak-yaroslav) [inline src/i18n/i18n.js]: same

File: src/components/Advantage/style.ts [hunk 25]
```diff
@@ -0,0 +1,23 @@
+import { Colors, media } from "@/ui";
+import styled from "@emotion/styled";
+
+export const AdvantageSC = styled.div`
+  border: 1px solid rgba(0, 0, 0, 0.2);
+  width: 100%;
+  max-width: 400px;
+  padding: 16px 160px 16px 32px;
+  ${media.DESKTOP} {
```
Comment (shumak-yaroslav) [inline src/components/Advantage/style.ts:10]: seporator

File: src/components/AdvantagesSection/style.ts [hunk 26]
```diff
@@ -0,0 +1,68 @@
...
+  max-width: 1920px;
+  width: 100%;
+  padding: 76px 120px;
+  display: flex;
+  flex-direction: column;
+  row-gap: 36px;
+  ${media.DESKTOP} {
+    padding: 76px 80px;
+  }
+  ${media.LAPTOP} {
```
Comment (shumak-yaroslav) [inline src/components/AdvantagesSection/style.ts:23]: same

File: src/components/SliderMeduim/SliderMedium.tsx [hunk 27]
```diff
@@ -0,0 +1,69 @@
...
+    <>
+      <ButtonsSlider swiperRef={swiperRef} />
+      <ContainerSwiperSC>
+        <CustomSwiperSC
+          slidesPerView={1}
+          onBeforeInit={(swiper) => {
+            swiperRef.current = swiper;
+          }}
+          loop={true}
+          breakpoints={{
```
Comment (shumak-yaroslav) [inline src/components/SliderMeduim/SliderMedium.tsx]: separate object for breakpoints

**Pull request merged**